config/config.py

```diff
@@ -340,19 +346,52 @@ def build_config(
             "muon_sf_varation": "sf",  # "sf" is nominal, "systup"/"systdown" are up/down variations
         },
     )
+    # electron scale factors configuration
+    configuration.add_config_parameters(
+        ["et", "ee", "em"],
+        {
+            "ele_sf_file": EraModifier(
+                {
+                    "2016preVFP": "data/jsonpog-integration/POG/EGM/2016preVFP_UL/electron.json.gz",
+                    "2016postVFP": "data/jsonpog-integration/POG/EGM/2016postVFP_UL/electron.json.gz",
+                    "2017": "data/jsonpog-integration/POG/EGM/2017_UL/electron.json.gz",
+                    "2018": "data/jsonpog-integration/POG/EGM/2018_UL/electron.json.gz",
+                }
+            ),
+            "ele_id_sf_name": "UL-Electron-ID-SF",
+            "ele_sf_year_id": EraModifier(
+                {
+                    "2016preVFP": "2016preVFP",
+                    "2016postVFP": "2016postVFP",
```

Comment:
Is it possible now to split 2016? Then we should also add this for the muon and tau SF configs.